stdlib/public/core/Duration.swift

```diff
@@ -0,0 +1,228 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, macCatalyst 9999, *)
+@frozen
+public struct Duration: Sendable {
+  @frozen
+  @usableFromInline
+  struct Bits: Sendable, Codable, Hashable, Equatable {
+    @inlinable
+    static var nanosecondMask: UInt64 { 0x000000003FFFFFFF }
+    
+    @inlinable
+    static var secondsSignFlags: UInt64 { 0x8000000000000000 }
+    
+    @inlinable
+    static var nanosecondsSignFlags: UInt64 { 0x8000000000000000 }
+
+    @usableFromInline
+    var high: UInt64
+    
+    @usableFromInline
+    var low: UInt64
+    
+    @usableFromInline
+    init(high: UInt64, low: UInt64) {
+      self.high = high
+      self.low = low
+    }
+  }
+  
+  @usableFromInline
+  var bits: Bits
+  
+  @inlinable
```

Comment:
Why do we need this (and the other conversion properties/methods below) to be inlinable?